showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITDynamicRoutingHeaders.java

```diff
@@ -114,16 +121,36 @@ public void onClose(int statusCode, HttpJsonMetadata trailers) {
 
   private HttpJsonCapturingClientInterceptor httpJsonInterceptor;
   private GrpcCapturingClientInterceptor grpcInterceptor;
-
+  private GrpcCapturingClientInterceptor grpcComplianceInterceptor;
   private EchoClient grpcClient;
   private EchoClient httpJsonClient;
+  private ComplianceClient grpcComplianceClient;
+
+  // Create grpcComplianceClient with Interceptor
+  public static ComplianceClient createGrpcComplianceClient(List<ClientInterceptor> interceptorList)
+      throws Exception {
+    ComplianceSettings grpcComplianceSettings =
+        ComplianceSettings.newBuilder()
+            .setCredentialsProvider(NoCredentialsProvider.create())
+            .setTransportChannelProvider(
+                ComplianceSettings.defaultGrpcTransportProviderBuilder()
+                    .setChannelConfigurator(ManagedChannelBuilder::usePlaintext)
+                    .setInterceptorProvider(() -> interceptorList)
+                    .build())
+            .build();
+    return ComplianceClient.create(grpcComplianceSettings);
+  }
```

Comment:
Can you move this to `TestClientInitializer` and invoke it via something like `TestClientInitializer.createGrpcComplianceClient(...)`?

Reply:
Done.